### PR TITLE
Add lack of progress reminder email

### PR DESCRIPTION
**Pull request by Adimote**
Add a reminder email to send to teams who haven't written code this calendar year.

### CODE REVIEW

**Comment by trickeydan** [comment]
I feel like this email is very much more likely to make teams drop out at the moment. Perhaps we could offer some sources of help to them? Details about tech days, forums, mentoring etc.

File: SR2019/2019-02-25-lack-of-progress-reminder [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+subject: Progress on your robot
+---
+
+Hey,
+
+We've noticed that your team hasn't made any changes to their code in the last couple of months, which is a possible indicator that your team may be struggling with making a robot.
+
+We always encourage you to push your team to get some kind of robot together, as a lot can be learned from the experience of putting together even the most basic robot, and it's a huge learning opportunity for them.
+
+You should try really really hard to make sure your team is making progress in preparation for the competition. However, we know there are situations where teams can't continue. In which case, there are some teams that would like to have a second kit. So please give us a shout and we can see if we can reallocate the kit to a team with two robots, but one kit.
```
Comment (RealOrangeOne) [inline SR2019/2019-02-25-lack-of-progress-reminder]: I don't think we need to be talking about the process of dropping out, it feels like we're guilt-tripping them into giving up.

**Comment by Tyler-Ward** [comment]
Did this email get merged in as I cant see it in the repo.

**Comment by RealOrangeOne** [comment]
it didn't, because I drafted it in MailChimp directly, which in hindsight was probably less than ideal. I'll force push it in (since it's already been sent).

**Pull request closed without merging**